docs/developer-guide/plugins.md

```diff
@@ -9,25 +9,25 @@ Each plugin is contained within its own JAR file, located according to the confi
 ## Configuration example
 
```

Comment:
Hi David. At the top of the file it claims that plugins can be "...implemented in any JVM language..". Well, that is only true if the JVM language can talk to Java. Because we don't have any particular language adapters besides Java I would remove this note from the document.